module-code/app/securesocial/controllers/MailTokenBasedOperations.scala

```diff
@@ -52,12 +39,26 @@ abstract class MailTokenBasedOperations extends SecureSocial {
    * @param isSignUp a boolean indicating if the token is used for a signup or password reset operation
    * @return a MailToken instance
    */
-  def createToken(email: String, isSignUp: Boolean): Future[MailToken] = {
+  def createToken(email: String, isSignUp: Boolean)(implicit env: RuntimeEnvironment): Future[MailToken] = {
```

Comment:
Maybe this should just take a `tokenDuration` arg that's passed in? That would make it more of a standalone helper.